lib/plugins/inventory.js

```diff
@@ -338,8 +338,8 @@ function inject (bot, { version, hideErrors }) {
     }
   }
 
-  async function openBlock (block) {
-    bot.activateBlock(block)
+  async function openBlock (block, direction, cursorPos) {
```

Comment:
looks like you forgot the default here

Reply:
Yeah I just noticed that there are also types missing